README.md

```diff
@@ -1,207 +1,339 @@
+| --testnet                            | starts LUKSO's testnet                                                                                                      |
+| --devnet                             | starts LUKSO's devnet                                                                                                       |
+| --geth -\*                           | \* Pass any flag to the Geth node [See docs for details](https://geth.ethereum.org/docs/fundamentals/command-line-options)  |
+| --erigon -\*                         | \* Pass any flag to the Erigon node [See docs for details](https://github.com/ledgerwatch/erigon)                           |
+| --prysm -\*                          | \* Pass any flag to the Prysm node [See docs for details](https://docs.prylabs.network/docs/prysm-usage/parameters)         |
+| --lighthouse -\*                     | \* Pass any flag to the Lighthouse node [See docs for details](https://lighthouse-book.sigmaprime.io/advanced-datadir.html) |
+| --validator -data-dir                | A path of validator's data directory (./validator_data)                                                                     |
+| --validator -verbosity               | verbosity for validator logs                                                                                                |
+| --validator -wallet-dir              | location of a generated wallet (./mainnet_keystore)                                                                         |
+| --validator -wallet-password-file    | location of password used for wallet generation (./config/mainnet/shared/secrets/validator-password.txt)                    |
+| --validator -chain-config-file       | path to config.yaml file (./config/mainnet/shared/config.yaml)                                                              |
+| --validator -monitor-host            | host used for interacting with Prometheus metrics (IP address)                                                              |
+| --validator -grpc-gateway-host       | host for gRPC gateway (IP address)                                                                                          |
+| --validator -rpc-host                | RPC server host (IP address)                                                                                                |
+| --validator -suggested-fee-recipient | address that receives block fees (0x12345..abcd)                                                                            |
+| --validator -output-dir              | directory where logs are created                                                                                            |
+| --validator -std-output              | set output to console                                                                                                       |
```

Comment:
Almost all of those flags are outdated, you can take a look in this ticket https://app.clickup.com/t/861mgjpbg to check which flags need to be defined by LUKSO CLI